cinema/views.py

```diff
@@ -0,0 +1,43 @@
+from cinema.serializers import MovieSerializer
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+
+from cinema.models import Movie
+
+
+@api_view(["GET", "POST"])
+def list_movie(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        else:
+            Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)

```

Comment:
There is a missing `return` statement before `Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)`. This will cause a syntax error. Add `return` to ensure the response is returned properly.